src/test/java/br/com/autonomiccs/apacheCloudStack/client/ApacheCloudStackClientTest.java

```diff
@@ -134,6 +136,24 @@ public void executeRequestTestExceptionWhenExecutingRequest() throws IOException
     }
 
     @Test
+    public void createRequestConfigDefaultTest() {
```

Comment:
Test cases are always awesome, but I will be picky one more time.

Lines 140 – 143 and 150-153 are basically the same, the only difference is the timeout value.
What about extracting them for a single method that receives the expected timeout value as a parameter?